Maths/TwinPrime.js

```diff
@@ -0,0 +1,43 @@
+function isPrime (n) {
+  let prime = false
```

Comment:
To add to what rak said, this variable is named misleadingly and the below sieve is pretty poor (O(n)) compared to other algos.

Reply:
What's the name of the algorithm already implemented for primes?

Reply:
Is it Maths/PrimeCheck?

Reply:
Yeah, you can use that one.